### ISSUE TITLE
Deadlink in Media section

### ISSUE THREAD

**Issue by Nerzal** [tinygo-org/tinygo-site#120]
The following link leads me to an error page:

https://videoh.infoq.com/presentations/20-mar-tinygosmallisgoingbig.mp4

The error message above the xml in the following graphic roughly translates to "there are not style sheets connected to this element. this is why the tree view is being used"

![grafik](https://user-images.githubusercontent.com/9110370/95023229-24520b00-067c-11eb-8bcc-5fa45e95cc8c.png)

In short the link seems to be dead. 
Can we replace the link with an active one? :)

**Comment by aykevl** [comment]
It looks like this was fixed by #123. If not, please comment here.
